hoomd/md/EvaluatorPairExpandedLJ.h

```diff
@@ -180,8 +178,12 @@ class EvaluatorPairExpandedLJ
 
             if (energy_shift)
                 {
-                Scalar rcut2inv = rcutinv * rcutinv;
-                Scalar rcut6inv = rcut2inv * rcut2inv * rcut2inv;
+                Scalar rcut = fast::sqrt(rcutsq);
+                Scalar r_cut_shifted = rcut - delta;
+                Scalar r_cut_shifted_inv = Scalar(1.0) / r_cut_shifted;
+
+                Scalar r_cut2inv = r_cut_shifted_inv * r_cut_shifted_inv;
+                Scalar rcut6inv = r_cut2inv * r_cut2inv * r_cut2inv;
                 pair_eng -= rcut6inv * (lj1 * rcut6inv - lj2);
```

Comment:
```suggestion
                Scalar r_cut = fast::sqrt(rcutsq);
                Scalar r_cut_shifted = r_cut - delta;
                Scalar r_cut_shifted_inv = Scalar(1.0) / r_cut_shifted;

                Scalar r_cut2_inv = r_cut_shifted_inv * r_cut_shifted_inv;
                Scalar r_cut6_inv = r_cut2_inv * r_cut2_inv * r_cut2_inv;
                pair_eng -= r_cut6_inv * (lj1 * r_cut6_inv - lj2);
```
Consistent (and more) underscoring